README.md

```diff
@@ -56,11 +70,39 @@ In this order:
   ```sh
   git clone git@github.com:wc-phdc-hia/testing.git
   ```
+
+- To get the latest Playwright run:
+  
+  ```sh
+  npm install -D @playwright/test@latest
+  ```
+
+- A package for setting environment variables:
+  
+  ```sh
+  npm install -D env-cmd
+  ```
   
+>**Important:** Create a .env file in the root folder that will contain your local SPV username and password. i.e. USERNAME="username here", PASSWORD="password here"
+
 ## Resources
 
 [Playwright](https://playwright.dev/docs/intro)
 
+## Testing
+
+End to end tests use a library called [Playwright](https://playwright.dev/).
+To run a test suite run `npm run playwright-tests` or `npx playwright test <name-of-test-file>` to run a specific test file.
```

Comment:
Please remove all reference to `npx` and add an advisory against using. Please also can you stop using it.

Reply:
done